content/user/advanced-modules/PDFTemplates.adoc

```diff
@@ -0,0 +1,116 @@
+---
+Title: PDF-Templates
+Weight: 85
+---
+
+
+:toc:
+:toc-title: Table of content
+
+:experimental:
+
+:imagesdir: ./../../../../images/en/user
```

Comment:
Please always use an absolute path for `imagesdir`, according to guidelines.

And include the screenshots in a new commit to this branch, they're still missing...